app/forms/candidate_interface/degree_wizard.rb

```diff
@@ -267,6 +267,7 @@ def attributes_for_persistence
           qualification_type: international_type,
           institution_name: university,
           subject: subject,
+          degree_subject_uuid: degree_subject_uuid,
```

Comment:
just to double check is this data available to us from the gem?

Reply:
Yes, so if they selected Animation in the subject autocomplete, it would first search by name in the gem, and then derive the uuid which would be https://github.com/DFE-Digital/dfe-reference-data/blob/main/lib/dfe/reference_data/degrees.rb#L3428